_exercises-containers/07-BasicsOfContainers.md

```diff
@@ -1,7 +0,0 @@
----
-title: Basics of Containers and Orchestrations
----
-
-# Basics of Containers and Orchestrations
-
-
```

Comment:
Why was this deleted?

Reply:
I deleted it because we already have "Basics of containers" in the Lectures section.